luigi/rpc.py

```diff
@@ -141,7 +145,7 @@ def _request(self, url, data, log_exceptions=True, attempts=3, allow_null=True):
         body = {'data': json.dumps(data)}
 
         for _ in range(attempts):
-            page = self._fetch(url, body, log_exceptions, attempts)
```

Comment:
I don't deepdive into the code, but when the central scheduler is not working the logging seems to explain a retry of 3